mock/breez-sdk/src/lib.rs

```diff
@@ -808,8 +859,8 @@ impl BreezServices {
         let now = Utc::now().timestamp();
 
         let amount_msat = req.amount_sat * 1_000;
-        let fee_msat = *LN_BALANCE_MSAT.lock().unwrap() - amount_msat;
-        *LN_BALANCE_MSAT.lock().unwrap() -= amount_msat + fee_msat;
+
+        send_payment_mock_channels(amount_msat + 1234);
```

Comment:
The fee `1234 msat` is being used in 3 places, maybe let's make it a constant for easier understanding

Reply:
I replaced it with randomly generated values instead :)